Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Builder
+FROM gradle:jdk17-alpine as builder
+
+RUN apk add --update nodejs npm
+
+WORKDIR /app
+COPY ./ /app/
+
+RUN gradle jar --no-daemon
+
+# Fetch Data
+FROM bitnami/git:2.43.0-debian-11-r1 as data
+
+ARG DATA_REPOSITORY=https://gitlab.com/YuukiPS/GC-Resources.git
+ARG DATA_BRANCH=4.0
+
+WORKDIR /app
+
+RUN git clone --branch ${DATA_BRANCH} --depth 1 ${DATA_REPOSITORY}
+
+# Result Container
+FROM amazoncorretto:17-alpine
+
+WORKDIR /app
+
+# Copy built assets
+COPY --from=builder /app/grasscutter-1.7.4.jar /app/grasscutter.jar
```

Comment:
this doesn't seem good; hardcoding the version of the jar (which is bound to change) will break this in the future

Reply:
Fixed by replacing the version with a wildcard. Assumes that their will only be one file produced that matches `grasscutter-*.jar`.